src/components/TodoComponents/Todo.js

```diff
@@ -1,14 +1,34 @@
 import React from 'react'
 
-const Todo = props =>{
+export default function Todo(props){
+
+    let classNames = 'todo';
+    if (props.todo.completed){
+        classNames = 'completed'; //for whatever reason todo.completed doesnt work
+    }
+
+    function updateCompleted(){
+
+        props.toggleComplete(props.todo.id);
+    }
```

Comment:
You could also just define this function inside `TodoList.js` (or even better, in `App.js) and then pass it as a prop for the `<Todo />` component (this way, Javascript will only have to create an instance of this method once).

Also, `updateCompleted` can be quite an ambiguous name for this function - perhaps `toggleCompleted` or `toggleCompletedStatus` might be more fitting.